config/application.rb

```diff
@@ -26,6 +26,11 @@ class Application < Rails::Application
 
     config.active_record.belongs_to_required_by_default = false
 
+    config.active_record.encryption.support_unencrypted_data = true
```

Comment:
Should we add a todo comment on this to remove it once we’ve encrypted all the records?